examples/datasets/plot_spm_faces_dataset.py

```diff
@@ -5,13 +5,14 @@
 ==========================================
```

Comment:
And you'll be happy to know that the source localization was fixed by using `method='shrunk'` for the cov :)